build/manifest-templates/clusterserviceversion.yaml

```diff
@@ -19,6 +19,16 @@ metadata:
     olm.skipRange: ">=1.0.2 <__SEMANTIC_VERSION__"
     operators.openshift.io/infrastructure-features: '__OLM_FEATURES__'
     __OLM_SUBSCRIPTION_ANNOTATION__
+    features.operators.openshift.io/disconnected: __OLM_FEATURE_DISCONNECTED__
```

Comment:
The `operators.openshift.io/infrastructure-features` annotation is quoted, whereas `features.operators.openshift.io/disconnected` isn't. In other words, the `OLM_FEATURES` variable does not contain quotes, whereas  `OLM_FEATURE_DISCONNECTED` does. 

I suggest adding the quotes here instead of having them in `OLM_FEATURE_DISCONNECTED`.

Same for all other variables introduced in this PR.

Reply:
Thank guys fixed in the next commit